deployment-scanner/src/main/java/org/jboss/as/server/deployment/scanner/FileSystemDeploymentService.java

```diff
@@ -147,6 +149,13 @@
     private final String relativePath;
     private final PropertyChangeListener propertyChangeListener;
 
+    void rollBack(List tasks, ModelNode failure) {
+        List<ScannerTask> scannerTasks = (List<ScannerTask>) tasks;
+        for(ScannerTask task : scannerTasks) {
+            task.handleFailureResult(failure);
+        }
```

Comment:
This isn't a rollback. It just puts unrelated failure info in the deployments dir. The ManualScanCallable is still running.